libraries/AP_SmartAudio/AP_SmartAudioDevice.cpp

```diff
@@ -0,0 +1,73 @@
+#include <AP_HAL/AP_HAL.h>
+#include "AP_SmartAudioDevice.h"
+
+AP_SmartAudioDevice::AP_SmartAudioDevice(AP_HAL::UARTDriver* _port, bool _use_trailing_zero) :
+    port(_port),
+    use_trailing_zero(_use_trailing_zero)
+{
+}
+
+void AP_SmartAudioDevice::update()
+{
+    switch(state) {
+        case State_READY:
+            if (update_power() || update_channel())
+                state = State_READING_RESPONSE;
```

Comment:
I think it would be better to follow the ArduPilot coding rules.
The processing of the if statement makes the block clear with "{", "}".

http://ardupilot.org/dev/docs/style-guide.html

Reply:
Tools/CodeStyle/ardupilot-astyle.sh semi-automaticly fix this and probably some other formatting things.
I would suggest to format the whole pr with it.

Reply:
fixed